src/js/sticky-header.js

```diff
@@ -0,0 +1,25 @@
+//Fixed navbar
```

Comment:
Could there be more comments in this file? File header comments for sure but also inline comments would be use too :)